### PR TITLE
FitTracker View (FXML, CSS)

### PR DESCRIPTION
**Pull request by KyletheWolff**
Here is the design for the view.

No controller has been added yet.

### CODE REVIEW

**Comment by KyletheWolff** [comment]
#16

**Comment by BrandonVanderMey** [comment]
One problem I see is that your FXML doesn't start with <fx:root /> tags.  If I remember correctly the reason we did that was so we can insert it into another fxml frame.

**Comment by KyletheWolff** [comment]
Oh crap I did forget root. I can add that in later this week

**Comment by KyletheWolff** [comment]
Added the root tag

File: src/com/kandbnutrition/views/FitTrackerView.fxml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<?import javafx.scene.shape.*?>
+<?import javafx.scene.text.*?>
+<?import javafx.geometry.*?>
+<?import java.lang.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.layout.AnchorPane?>
+
+<fx:root type="javafx.scene.layout.AnchorPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<AnchorPane prefHeight="506.0" prefWidth="500.0" stylesheets="@../resource/FitTrackerView.css" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (BrandonVanderMey) [inline src/com/kandbnutrition/views/FitTrackerView.fxml]: The <AnchorPane> is unneeded since you already defined an anchor pane at the <root> tag.

**Comment by KyletheWolff** [comment]
I will need the imports for the rest of the elements.

**Comment by BrandonVanderMey** [comment]
Say that again?

**Comment by KyletheWolff** [comment]
Are you talking about the Anchorpane?

**Comment by BrandonVanderMey** [comment]
Yes, the <root> tag type is "AnchorPane"  and beneath it is an <AnchorPane> tag.  I don't think that is needed unless somewhere in the code you plan (or already have) two Anchor Panes side by side in the <root> tag.

**Comment by KyletheWolff** [comment]
Edited

**Comment by BrandonVanderMey** [comment]
Was there any attributes from the <AnchorPane> tag you removed that you needed?

**Comment by KyletheWolff** [comment]
Ya I did it

**Comment by BrandonVanderMey** [comment]
Okay :) Good job. Merged

**Pull request merged**